plugins/gnostic-go-generator/render_client.go

```diff
@@ -22,14 +22,19 @@ import (
 )
 
 // ParameterList returns a string representation of a method's parameters
-func ParameterList(parametersType *surface.Type) string {
+func (renderer *Renderer) ParameterList(parametersType *surface.Type) string {
+	if parametersType == nil ||
+		parametersType.Fields == nil ||
+		len(parametersType.Fields) == 0 {
```

Comment:
This isn't really a blocker, but would it be possible to make the following code work even when `len(Fields)` is `0`? Removing the need for this check. I like it when loop code works in 0 cases as well.

The thing I see preventing that is the extra `\n` appended to the resulting string...